back/app/Http/Controllers/MessageController.php

```diff
@@ -14,10 +14,17 @@ class MessageController extends Controller
     public function index()
     {
         $this->authorize('index', Message::class);
-        
+
         return response()->json(MessageResource::collection(Message::all()));
     }
 
+    public function display()
+    {
+        $this->authorize('display', Message::class);
+
+        return response()->json(MessageResource::collection(Message::active()->get()));
+    }
```

Comment:
J'aurais appelé la fonction **current** au lieu de **display** pour que ça soit consistant.
Fait aussi bien appel à la méthode `->first()` plutôt que `->get()` aussi

```suggestion
    public function current()
    {
        $this->authorize('current', Message::class);

        return response()->json(MessageResource::collection(Message::active()->first()));
    }
```